src/main/java/com/github/kaklakariada/fritzbox/model/User.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * A Java API for managing FritzBox HomeAutomation
+ * Copyright (C) 2017 Christoph Pirkl <christoph at users.sourceforge.net>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.github.kaklakariada.fritzbox.model;
+
+import org.simpleframework.xml.Attribute;
+
+public class User {
```

Comment:
According to the xml, the user also has a name:

```xml
    <Users>
        <User>UserA</User>
        <User>UserB</User>
        <User last="1">UserC</User>
    </Users>
```
Please add a field for the name.

Reply:
implemented

Reply:
implemented